Mathlib/Algebra/Module/Zlattice.lean

```diff
@@ -317,6 +316,12 @@ protected theorem isAddFundamentalDomain [Finite ι] [MeasurableSpace E] [OpensM
     fun x => exist_unique_vadd_mem_fundamentalDomain b x
 #align zspan.is_add_fundamental_domain Zspan.isAddFundamentalDomain
 
+theorem measure_fundamentalDomain_ne_zero [Finite ι] [MeasurableSpace E] [BorelSpace E]
+    {μ : Measure E} [Measure.IsAddHaarMeasure μ] (hμ : μ ≠ 0) :
```

Comment:
You should remove h\mu here, as a Haar measure is always nonzero (it gives positive mass to open sets). Same thing below.

Reply:
I removed the hypothesis from the lemma in `Zlattice.lean`, but I am not sure what you mean by `Same thing below`. Do I need to change also the statement of the lemma in `FundamentalDomain.lean`? If I removed the hypothesis in this lemma, it does not work anymore as it is.

Reply:
I read too quickly, what you did is perfect.